extensions/db/src/main/java/org/mule/extension/db/api/param/ParameterizedStatementDefinition.java

```diff
@@ -38,8 +40,8 @@
    */
   @Parameter
   @Optional
-  //@Content TODO: MULE-10641 re-enable datasense for these parameter
   @Placement(order = 2)
+  @TypeResolver(BaseDbMetadataResolver.class)
```

Comment:
The `Base` prefix denotes an abstract class meant to be extended. This is something you're actually using so rename